### DISCUSSION TITLE
How to send video with thumb

### DISCUSSION THREAD

**Discussion by wellmoon** [go-telegram-bot-api/telegram-bot-api#494] [Q&A]
I want send video with thumb, but there is no field thumb in NewVideoUpload

**Comment by Syfaro** [comment]
Hi, 

The previous version of the library, v4, did not support multiple file uploads. v5 now supports this. You can do something like the following: 

```go
video := tgbotapi.NewVideo(chatID, tgbotapi.FilePath("video.mp4"))
video.Thumb = tgbotapi.FilePath("thumb.mp4")

msg, err := bot.Send(video)
```
